tests/appsec/rasp/__init__.py

```diff
@@ -0,0 +1,42 @@
+def validate_rasp_attack(span, rule, parameters=None):
+    assert "_dd.appsec.json" in span["meta"], "'_dd.appsec.json' not found in 'meta'"
+
+    json = span["meta"]["_dd.appsec.json"]
+    assert "triggers" in json, "'triggers' not found in '_dd.appsec.json'"
+
+    triggers = json["triggers"]
+    assert len(triggers) == 1, "multiple appsec events found, only one expected"
+
+    trigger = triggers[0]
+    obtained_rule_id = trigger["rule"]["id"]
```

Comment:
You do nothing with this value, is it expected ?

Reply:
No, I just forgot to use it in the assertion below

Reply:
ok, taking care of it